src/benchmark_environments/mujoco.py

```diff
@@ -11,6 +11,8 @@
     walker2d_v3,
 )
 
+import benchmark_environments.util as util
```

Comment:
```suggestion
from benchmark_environments import util
```
feels slightly more idiomatic to me